src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -6,6 +6,40 @@ description: Overview and specification of the hardware used by NUbots.
+The NUbots team uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
 
-The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/). The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm). The robots use two [FLIR Blackfly S](https://flir.app.boxcn.net/s/x0r0bdsjstdkudjd2dwlxoa5fdvj2gpg/file/418603726810) cameras positioned in their head. In the torso lies the Intel NUC7i7bnh, 8GB RAM, 256GB SSD, Robotis CM740 subcontroller, and a 3850mAh 4s LiPo battery.
+![Coronal and saggital images of the NUgus robot](../../../images/nugus.png 'Coronal and saggital images of the NUgus robot')
+
+## igus® Modifications
+
+The following modifications were made to fit our specific requirements:
+
+- The components of the robot were split to to fit the size of the printer bed and volume.
+- The torso was modified to have open shoulder covers for shoulder servo cable access.
+
+## Servos
+
+The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm), with two in each ankle, three in each hip, and one in each knee.
+
+There are custom made mounting brackets milled from aluminium for the flywheels and neck. There are 3D printed cable clamps on the back of the knees, angles and hip blocks. The NUgus robot uses servo-to-servo mounting brackets.
+
+## Cameras
+
+The NUgus robot uses two [FLIR Blackfly S](https://www.flir.com/products/blackfly-s-usb3?model=BFS-U3-13Y3C-C) USB3 cameras with a C-Mount lens positioned in their head, producing images at 170FPS. Two custom-built S-Mount to C-Mount lens adaptors are used to attach two [Lensation 180° fisheye S-Mount lenses](https://www.lensation.de/product/BF10M19828S118C/). Two [20" USB3 micro USB to up angle USB A cables](https://www.angledcables.com/parts/rr-aums-xxgr.html#RR-AUMS-20GR) are used to connect the cameras to the robot's computer.
+
+## Computing
+
+The NUgus robot uses the [Intel NUC7i7bnh](https://www.pccasegear.com/products/39044/intel-boxnuc7i7bnh-core-i7-7th-gen-nuc-barebone-kit) with a [Silverstone NT07-115X](https://www.silverstonetek.com/product.php?pid=442&area=en) cooling fan. The cooling fan's supporting arms were removed and extra skrew holes drilled to line up with the mounting points on the NUC. A copper spacing plate was milled to bridge the gap between the NUC processor and the fan.
```

Comment:
Should we be giving links to product pages like pccasegear, or should we be giving links to the manufacturer's pages? (e.g. https://www.intel.com.au/content/www/au/en/products/boards-kits/nuc/kits/nuc7i7bnh.html)